src/pages/data-entry/data-entry.jsx

```diff
@@ -1,127 +1,214 @@
-import React, { Component } from "react";
+import React, { Component } from 'react'
+import { useQuery } from '@apollo/react-hooks'
+import { useMutation } from '@apollo/react-hooks'
+import { gql } from 'apollo-boost'
+
+const units = gql`
+  {
+    getAllAvailableUnits
+    }
+  }
+`
+
+const insertMutation = gql`
+  mutation InsertMutation(
+    $unit: String!
+    $value: Float!
+    $latitude: Float!
+    $longitude: Float!
+  ) {
+    insertValue(
+      unit: $unit
+      value: $value
+      latitude: $latitude
+      longitude: $longitude
+    )
+  }
+`
+
+function DataFunction () {

```

Comment:
To make it easier on anyone reviewing or looking at this code later, lets keep to coding conventions, and also we already know its a function. DataFunction -> getAllUnitsQuery.